ooni/deck/store.py

```diff
@@ -21,6 +21,28 @@ class InputNotFound(Exception):
 class DeckNotFound(Exception):
     pass
 
+
+def write_txt_from_csv(in_file, out_file, func, skip_header=True):
+    out_fh = out_file.open('w')
```

Comment:
At this point there is actually no real valid reason for not doing what you suggest. I guess I got into the bad habbit of doing the above pattern to maintain backward compatibility with python 2.6 that didn't support multiple with statements on the same line.

I will adjust accordingly.